pkg/apis/networking/types.go

```diff
@@ -158,6 +158,9 @@ type NetworkPolicy struct {
 	// Priority represents the relative priority of this Network Policy as compared to
 	// other Network Policies. Priority will be unset (nil) for K8s Network Policy.
 	Priority *float64
+	// TierPriority represents the priority of the Tier associated with this Network
```

Comment:
NetworkPolicy is a word, and should be in one line?

Reply:
yes will update this one along with change to Platform